City-of-Helsinki/drupal-helfi-platform-test
# UHF-8381: Platform v3 update

# [UHF-8381](https://helsinkisolutionoffice.atlassian.net/browse/UHF-8381)

Update to platform config v3. Test on a 

## What was done
* Updated `helfi_platform_config` to v3 and all configurations based on new base config
* Updated Guzzle to version 7
* Updated platforms tools to latest versions

## How to install and test
* Read the platform v3 update documentation
   * https://github.com/City-of-Helsinki/drupal-helfi-platform-config/blob/3.x/documentation/update.md#update-instructions
* Checkout this branch 
   * `git checkout UHF-8381-platform-v3-update`
* Run `make new`
* Click around the site and make sure normal content creation works
